config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  get '/sweets', to: "sweets#index", as: "sweets"
-  get '/vendors', to: "vendors#index", as: "vendors"
-
+  get '/sweets', to: 'sweets#index', as: 'sweets'
+  get '/vendors', to: 'vendors#index', as: 'vendors'
+  resources :sweets, :vendors, :vendor_sweets, only: %i[index show new create edit update destroy]
```

Comment:
You `only` here is only all the routes, so you can omit it. 
```suggestion
  resources :sweets, :vendors, :vendor_sweets
```
HOWEVER, since you're not using all the routes, it would be best to only define the routes that you're actually using.